### PR TITLE
Updated importlib and cache imports for Django 1.7+

### PR DESCRIPTION
**Pull request by SimonSteinberger**
This code should work across all relevant Django and Python versions.

### CODE REVIEW

File: ratelimit/utils.py [hunk 1]
```diff
@@ -4,9 +4,16 @@
 import zlib
 
 from django.conf import settings
-from django.core.cache import get_cache
+try:
+    from django.core.cache import caches
+except ImportError:
+    from django.core.cache import get_cache as caches
```
Comment (jsocol) [inline ratelimit/utils.py]: We can avoid the awkward, and too-long, line at 137 by reversing this, and doing something like

```python
try:
    from django.core.cache import caches
    def get_cache(name):
        return caches[name]
except ImportError:
    from django.core.cache import get_cache
```
Then just calling `get_cache` on 137 (and anywhere else we'd need to in the future).

**Comment by jsocol** [comment]
I don't know _what_ happened with Travis the first time, the job took over 7 hours before I killed it.

Along with the one comment, which will fix Travis' PEP8 complaint, would you mind squashing this into one commit (or two, one fixing the `importlib` import and one fixing the `caches` import)? Thanks!

**Comment by jsocol** [comment]
Oh, I guess [this](http://www.traviscistatus.com/incidents/vbgbxllvy976) is what happened to Travis.

**Comment by SimonSteinberger** [comment]
Oh yeah, that's much better :-) Like it!

**Comment by jsocol** [comment]
This is fixed in [`ratelimit/compat.py`](https://github.com/jsocol/django-ratelimit/blob/master/ratelimit/compat.py)

**Pull request closed without merging**